integration-testing/rnode_testing/rnode.py

```diff
@@ -44,22 +53,22 @@ def __repr__(self):
 
 
 class TimeoutError(Exception):
-    def __init__(self, command, timeout):
+    def __init__(self, command: Iterator, timeout: int) -> None:
```

Comment:
Why `Iterator` and not `Tuple[...]`?

Reply:
It should be `Union[Tuple[str, ...], str]`